runtime/kusama/src/lib.rs

```diff
@@ -318,7 +318,7 @@ parameter_types! {
 	// 28 eras for unbonding (7 days).
 	pub const BondingDuration: staking::EraIndex = 28;
 	// 28 eras in which slashes can be cancelled (7 days).
```

Comment:
```suggestion
	// 27 eras in which slashes can be cancelled (~7 days).
```

And in the other places too